app/views/dishes/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<h1>Today's Dishes</h1>
+
+<h3><%= @dish.name %><br> description: <%= @dish.description %></h3>
+
+<h3>Ingredients</h3>
+  <div id="dish-<%= @dish.id %>-ingredients">
+    <% @dish.ingredients.each do |ingredient|%>
+      Name: <%= ingredient.name %>
+      Calories: <%= ingredient.calories %>
```

Comment:
For this US, we were looking for you to do a calculation of all calories in a dish. So, you could add a model method to the Dish model like so that would sum all the ingredients' calories together
```
def total_calories
   self.ingredients.sum(:calories)
end
```